Core/Core/Credentials/AccountManager.cs

```diff
@@ -106,7 +106,7 @@ public static Account GetDefaultAccount()
         var firstAccount = GetAccounts().FirstOrDefault();
         if (firstAccount == null)
         {
-          Log.CaptureAndThrow(new SpeckleException("No Speckle accounts found. Visit the Speckle web app to create one."), level: Sentry.Protocol.SentryLevel.Info);
+          Log.CaptureException(new SpeckleException("No Speckle accounts found. Visit the Speckle web app to create one."), level: Sentry.Protocol.SentryLevel.Info);
```

Comment:
@didimitrie tests pass, but just to confirm this is ok and that `GetDefaultAccount()` returning null doesn't break anything